swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -215,54 +197,89 @@ Follow the instructions given in this section to develop the service.
-        resource function post categories/[string category]/reserve(ReservationRequest payload) 
-                returns http:Created|http:NotFound|http:InternalServerError {
-            
+    - Open the [Ballerina HTTP API Designer](https://wso2.com/ballerina/vscode/docs/design-the-services/http-api-designer) in VS Code.
+
+    - Use `/healthcare` as the service path (or the context) for the service attached to the listener that is listening on port `8290`.
+
+        ![Define the service](/learn/images/integration-tutorials/sending-emails-from-a-service/define_a_service.gif)
+     
+    - Define an HTTP resource that allows the `POST` operation on the resource path `/categories/{category}/reserve` and accepts the `category` path parameter (corresponding to the specialization). Use `ReservationRequest` as a parameter indicating that the resource expects a JSON object corresponding to `ReservationRequest` as the payload. Use `http:Created`, `http:NotFound`, and `http:InternalServerError` as the response types.
+
+        ![Define the resource](/learn/images/integration-tutorials/sending-emails-from-a-service/define_a_resource.gif)
+
+        The generated service will be as follows.
+
+        ```ballerina
+        service /healthcare on new http:Listener(8290) {
+            resource function post categories/[string category]/reserve(ReservationRequest payload) 
+                    returns http:Created|http:NotFound|http:InternalServerError {
+                
+            }
         }
-    }
+        ```
+
+5. Define [configurable variables](https://ballerina.io/learn/by-example/#configurability) for the URLs of the backend services. Also, define configurable variables for the host, username, and password of the SMTP client.
+
+    ![Define a configurable variable](/learn/images/integration-tutorials/sending-emails-from-a-service/define_a_configurable_variable.gif)
+
+    The generated code will be as follows.
+
+    ```ballerina
+    configurable string hospitalServicesBackend = "http://localhost:9090";
+    configurable string paymentBackend = "http://localhost:9090/healthcare/payments";
+    configurable string host = "smtp.gmail.com";
+    configurable string username = ?;
+    configurable string password = ?;
     ```
 
-   - Use `/healthcare` as the service path (or the context) for the service attached to the listener that is listening on port `port`.
-     
-   - The HTTP resource allows the `POST` operation on resource path `/categories/{category}/reserve`, where `category` (corresponding to the specialization) is a path parameter.
+    > **Note:** Enable two factor authentication on your Google account, generate an app password, and use the app password in place of your email password. The app password can be generated with [this link](https://myaccount.google.com/apppasswords?pli=1&rapt=AEjHL4Mf5XDD4rE79YJpP5E2NoNwhvXMET_TWyBcQRn-HMzt0PI8BmptpMGRiBVIamW-0ECgVZtXxMRA19bL4Wfnq_hmjBEMqA).
```

Comment:
Using `Config.toml` part is not explained. IMO, the user can't find a way to use them in the tutorial.